lib/phoenix_live_view/channel.ex

```diff
@@ -820,6 +821,27 @@ defmodule Phoenix.LiveView.Channel do
     end
   end
 
+  defp check_patch_redirect_limit!(state) do
```

Comment:
We cannot keep it in the state? Do we have to use pdict?

Reply:
We can. I wasn't sure if we wanted it "hidden" from the socket in userland though. Maybe we can store it in `private`?